inc/zoo/swar/math.h

```diff
@@ -0,0 +1,75 @@
+#pragma once
+#include "SWAR.h"
+#include <cstddef>
+#include <cstdint>
```

Comment:
`SWAR.h` includes this, or its equivalent, `stdint.h`, I see no point in things like `std::uint64_t` instead of just `uint64_t`